internal/node.bzl

```diff
@@ -152,15 +175,6 @@ _NODEJS_EXECUTABLE_ATTRS = {
         allow_files = True,
         single_file = True),
     "node_modules": attr.label(
-        # We expect most users declare a binary/test and run it within the same
```

Comment:
Why remove this comment? Maybe you don't stand by its recommendation "should override", but I think it is still worth documenting what the attr node_modules is, why it is tricky to pick a "correct" value in presence of npm hoisting.

Reply:
Updated the comment rather than remove it.